### PR TITLE
Fixing Event Callback schemas and examples

### PR DESCRIPTION
**Pull request by jtreminio-dropbox**

### CODE REVIEW

**Review by fenghao-chen — COMMENTED** [review]
Looks like EventCallbackHelper.mustache for DotNet is missing?

File: sdks/java/templates/EventCallbackHelper.mustache [hunk 1]
```diff
@@ -2,23 +2,47 @@
...
     /**
     * Verify that a callback came from HelloSign.com
     *
     * @param apiKey
-    * @param event
+    * @param event_callback
     * @return a boolean value indicating whether the callback event is valid
     */
-    public static boolean isValid(String apiKey, EventCallbackRequestEvent event) {
+    public static boolean isValid(String apiKey, EventCallbackRequest event_callback) {
```
Comment (fenghao-chen) [inline sdks/java/templates/EventCallbackHelper.mustache:25]: nit: `event_callback` should be `eventCallback` (camel case)

**Comment by jtreminio-dropbox** [comment]
> Looks like EventCallbackHelper.mustache for DotNet is missing?

It's just the cs file directly:

* https://github.com/hellosign/hellosign-openapi/blob/event-callback-fixes/sdks/dotnet/openapi-config.yaml#L17-L20
* https://github.com/hellosign/hellosign-openapi/blob/event-callback-fixes/sdks/dotnet/templates/EventCallbackHelper.cs

**Review by fenghao-chen — APPROVED** [review]
LGTM

**Pull request merged**